src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -180,6 +183,21 @@ void MusicBrainzRecordingsTask::doNetworkReplyFinished(
     // Continue with next recording id
     DEBUG_ASSERT(!m_queuedRecordingIds.isEmpty());
     emit currentRecordingFetched();
+    auto inhibitTimerElapsed = m_inhibitTimer.elapsed(true);
+    qDebug() << "Task took:" << inhibitTimerElapsed.toIntegerMillis();
+    qDebug() << kMinTimeBetweenMbRequests.toIntegerMillis();
+    if (inhibitTimerElapsed.toIntegerMillis() >= kMinTimeBetweenMbRequests.toIntegerMillis()) {
+        qDebug() << "Task took more than a second, slot is calling now.";
+        m_inhibitTimer.restart(true);
+        m_requestTimer.start();
```

Comment:
Here you need to use the minimum time. I guess `start(1)` or such.

Reply:
Done :+1: